image_client/PIC_upload_undo/undo_csv_import.py

```diff
@@ -0,0 +1,131 @@
+"""This file is home to the DatabasePurger, used with specific MD5 codes to purge upload batches to the
+    database.
+    NOTE: only use this process if your upload process includes a LOCK user command
+          to prevent other changes to the database during upload"""
+import image_client.picturae_config as picturae_config
+from image_client.importer import Importer
+import traceback
+
+
+class DatabasePurger(Importer):
```

Comment:
class name and file name should match, more or less. change this to be some variation of "undo"